addons/im_livechat/static/tests/new/discuss_tests.js

```diff
@@ -30,8 +30,8 @@ QUnit.test("No call buttons", async function (assert) {
     });
     const { openDiscuss } = await start();
     await openDiscuss();
-    assert.containsNone(target, ".o-mail-discuss-actions button i.fa-phone");
-    assert.containsNone(target, ".o-mail-discuss-actions button i.fa-gear");
+    assert.containsNone(target, ".o-mail-discuss-header button[title='Start a Call']");
+    assert.containsNone(target, ".o-mail-discuss-header button[title='Show Call Settings']");
```

Comment:
assert on icon looked as legitimate to me a  assert on title (depending on whether we target standard or screen reader in the test)

Reply:
Almost as legitimate. The `fa-users` can be used twice, either as a command or the thread icon.
So I used `title`/`aria-label` instead because it's more reliable to target the specific element.